### PR TITLE
[APL-336] - Minor textual changes for APL feature

### PR DESCRIPTION
**Pull request by dennisvankekem**
## Description 📝
Minor textual changes for APL feature

## Changes  🔄
List any change relevant to the reviewer.
- Doc links have been changed from old/placeholders to new documentation pages . 
- APL Acronym removed from public facing text.
- Node pool threshold for APL now checks for vcpu as well. 

## Target release date 🗓️
11/12

### CODE REVIEW

**Review by jdamore-linode — APPROVED** [review]

**Review by bnussman-akamai — APPROVED** [review]

File: packages/manager/src/features/components/PlansPanel/constants.ts [hunk 1]
```diff
@@ -8,7 +8,8 @@ export const PLAN_NOT_AVAILABLE_IN_REGION_COPY =
   "This plan isn't available for the selected region.";
 export const PLAN_IS_CURRENTLY_UNAVAILABLE_COPY =
   'This plan is currently unavailable.';
-export const PLAN_IS_TOO_SMALL_FOR_APL_COPY = 'This plan is too small for APL';
+export const PLAN_IS_TOO_SMALL_FOR_APL_COPY =
+  'This plan is too small for Application Platform for LKE';
```
Comment (mjac0bs) [inline packages/manager/src/features/components/PlansPanel/constants.ts:12]:
```suggestion
  'This plan is too small for Application Platform for LKE.';
```
Nit: can we add a period here for consistency with the rest of our plan messaging?

File: packages/manager/src/features/Kubernetes/KubernetesClusterDetail/APLSummaryPanel.tsx [hunk 2]
```diff
@@ -50,7 +50,7 @@ const checkConsoleURL = async (
       if (error.response && error.response.status === 404) {
         setStatus({
           message:
-            'APL is still being deployed, please check back in a minute.',
+            'Installation still in progress, please check back in a minute.',
```
Comment (mjac0bs) [inline packages/manager/src/features/Kubernetes/KubernetesClusterDetail/APLSummaryPanel.tsx:53]:
```suggestion
            'Installation still in progress; please check back in a minute.',
```
Nit: these sentences are two independent clauses. Can we use a semicolon (or period) here to be [grammatically correct](https://owl.purdue.edu/owl/general_writing/punctuation/commas/commas_vs_semicolons.html)?

**Review by mjac0bs — APPROVED** [review]
Thanks @dennisvankekem - verified all the links look good. For the copy, I had two minor suggestions, but approving pending those.

**Comment by mjac0bs** [comment]
Thanks! I'll merge once CI finishes. 🚀

**Pull request merged**